[Bug] Prism.Unity.Uno 8.0.0.1693-ci doesn't work on UWP

### Description

I have tried Prism.Unity.Uno package that was added by the PR #2054 on an UWP project.
But I met build errors when first build after adding `Prism.Unity.Uno` package from MyGet.
The error messages are:
```
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(4643,5): error MSB3030: Could not copy the file "C:\Users\k_ota\.nuget\packages\prism.unity.uno\8.0.0.1693-ci\lib\uap10.0.16299\Prism.Uno\Services\Dialogs\DialogWindow.xaml" because it was not found.
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(4643,5): error MSB3030: Could not copy the file "C:\Users\k_ota\.nuget\packages\prism.unity.uno\8.0.0.1693-ci\lib\uap10.0.16299\Prism.Uno\Prism.Uno.xr.xml" because it was not found.
```

### Steps to Reproduce

1. Create a UWP project
2. Add Prism.Unity.Uno 8.0.0.1693-ci from 
3. Build the project

### Expected Behavior

No build error.

### Actual Behavior

There are two build errors:
```
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(4643,5): error MSB3030: Could not copy the file "C:\Users\k_ota\.nuget\packages\prism.unity.uno\8.0.0.1693-ci\lib\uap10.0.16299\Prism.Uno\Services\Dialogs\DialogWindow.xaml" because it was not found.
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(4643,5): error MSB3030: Could not copy the file "C:\Users\k_ota\.nuget\packages\prism.unity.uno\8.0.0.1693-ci\lib\uap10.0.16299\Prism.Uno\Prism.Uno.xr.xml" because it was not found.
```

### Basic Information

- Version with issue: Prism.Unity.Uno 8.0.0.1693-ci.
- Last known good version: None
- Xamarin.Forms version: None
- IDE: Visual Studio 2019 16.5.4

### Screenshots

![image](https://user-images.githubusercontent.com/79868/80196245-15d8df80-8658-11ea-95d9-c5097fdad883.png)

### Reproduction Link

The repro project link:
https://github.com/runceel/UnoPrismApp

--- a/src/Uno/Prism.Uno/Prism.Uno.csproj
+++ b/src/Uno/Prism.Uno/Prism.Uno.csproj
@@ -20,7 +20,6 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)'=='uap10.0.16299'">
     <PackageReference Include="Microsoft.Xaml.Behaviors.Uwp.Managed" Version="2.0.1" />
-    <PackageReference Include="CommonServiceLocator" Version="2.0.5" />
   </ItemGroup>
   <ItemGroup>
     <Page Include="**\*.xaml" Exclude="bin\**\*.xaml;obj\**\*.xaml" />

--- a/src/Uno/Prism.Uno/Prism.Uno.csproj
+++ b/src/Uno/Prism.Uno/Prism.Uno.csproj
@@ -8,6 +8,7 @@
     <DefineConstants>$(DefineConstants);HAS_WINUI</DefineConstants>
     <NoWarn>$(NoWarn);1591</NoWarn>
     <PackageId>Prism.Uno</PackageId>
+    <GenerateLibraryLayout>true</GenerateLibraryLayout>
   </PropertyGroup>
   <PropertyGroup Condition="'$(TargetFramework)'=='netstandard2.0'">
     <DefineConstants>$(DefineConstants);__WASM__</DefineConstants>

--- a/src/Uno/Prism.Uno/PrismApplicationBase.cs
+++ b/src/Uno/Prism.Uno/PrismApplicationBase.cs
@@ -1,5 +1,4 @@
-﻿using CommonServiceLocator;
-using Prism.Events;
+﻿using Prism.Events;
 using Prism.Ioc;
 using Prism.Logging;
 using Prism.Modularity;

